be/src/tools/segment_tool.h

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <gflags/gflags.h>
+
+#include "common/status.h"
+#include "gen_cpp/segment_v2.pb.h"
+#include "env/env.h" // RandomAccessFile
+
+namespace doris {
+
+class SegmentTool {
+public:
+    static void process();
```

Comment:
It will be better not to use `static` to leverage objective. The encapsulation is better than using static.